File: Algorithm/conditional_and_loops/number_pattern/C++/0001_solution.cpp
/**
 * solution of Problem 'Number pattern'.
 * @file 0001_solution.cpp
 * @author mohdishaq
 **/

#include<iostream>
using  namespace std;

int main() {
    int n;
    cin >> n;

    for (int row = 1; row <= n; row++) {
        int temp = row;
        for (int column = 1; column <= row; column++, temp++) {
            cout << temp <<" ";
        }
        cout << endl;
    }
    return 0;
}

File: Algorithm/conditional_and_loops/number_pattern/java/0001_solution.java
/**
 * solution of problem'number_pattern'.
 * @file 0001_solution.java
 * @author mohdishaq
 **/

import java.util.Scanner;

class Solution {

  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    int n = s.nextInt();
    int row = 1;
    while (row <= n) {
      int col = 1;
      int value = row;
      while (col <= row) {
        System.out.print(value + " ");
        value++;
        col++;
      }
      System.out.println();
      row++;
    }
  }
}

